docs/DafnyRef/Statements.14.expect

```diff
@@ -1,3 +1,3 @@
-text.dfy(5,5): Warning: /!\ No terms found to trigger on.
+text.dfy(13,5): Warning: /!\ No terms found to trigger on.
```

Comment:
I just realize how not maintainable it is that statements' tests are numerically ordered in the document. That will make inserting things harder, no?

Reply:
No -- the order does not matter, just needs to be unique. They are currently more or less ordered because I started at the top and progressed to the bottom.